app/views/candidate_mailer/nudge_unsubmitted_with_incomplete_courses.text.erb

```diff
@@ -4,7 +4,17 @@ You have not applied to any teacher training courses yet.
 
 [Sign into your account to apply for up to 4 courses](<%= candidate_magic_link(@application_form.candidate) %>).
 
-# Support with your application
+Popular courses fill up quickly and places are allocated as soon as people apply. Submit your application as soon as you are ready to avoid missing out on a limited number of course places.
+
+You may need to complete a [subject knowledge enhancement](<%= email_link_with_utm_params(t('get_into_teaching.url_subject_knowledge_enhancement'), 'nudge_unsubmitted_with_incomplete_courses', @application_form.phase) %>) (SKE) course and other conditions before starting your training.
+
+[Apply now to make sure you have enough time](<%= candidate_magic_link(@application_form.candidate) %>).
+
+Some subjects and courses have bursaries of up to £29,000 and scholarships of up to £31,000. These courses fill up more quickly than other courses.
+
+[Check if your subject has a bursary or scholarship](<%= email_link_with_utm_params t('get_into_teaching.url_scholarships_and_bursaries_funding_widget'), 'nudge_unsubmitted_with_incomplete_courses', @application_form.phase %>)
```

Comment:
Are we missing a full stop here? We have one for the link on line 11.

Reply:
Some links seem to have full stops, others don't. It would be good to clarify whether they should have full stops or not and be consistent going forward.

Reply:
Ok, if the links are at the end of a sentence, they should have a full stop. But if they are on their own, like this one, it should NOT. I will go through the changes and make changes where required.

Reply:
Fixed here https://github.com/DFE-Digital/apply-for-teacher-training/pull/10106/commits/db4d0c80c0e598b6e1298dc1299edbce12fbfdb3